alibi/explainers/similarity/backends/pytorch/base.py

```diff
@@ -0,0 +1,97 @@
+"""Torch backend for similarity explainers.
+
+Methods unique to the Torch backend are defined here. The interface this class defines syncs with the torch backend in
+order to ensure that the similarity methods only require to match this interface.
+"""
+
+import random
+from typing import Callable
+
+import numpy as np
+import torch.nn as nn
+import torch
+
+
+class _TorchBackend(object):
+    device = None
+
+    @staticmethod
+    def get_grads(
+            model: nn.Module,
+            x: torch.Tensor,
+            y: torch.Tensor,
```

Comment:
Although not a big deal, I would try to be consistent with the notation from all the explainers and use capital letters for tensors: x -> X, y -> Y.